Update `miner` -> `validator` to match alias.

kellybyrd opened the pull request:
While helping someone on Discord, he noticed the alias was being set at validator, but later commands try to use the alias miner.

kellybyrd committed:
Update `miner` -> `validator` to match alias.

While helping someone on Discord, he noticed the alias was being set at validator, but later commands try to use the alias miner.

PharkMillups commented:
@lthiery Did we get to the bottom of this?

lthiery commented:
The alias that I see in validator/monitor.mdx (https://github.com/helium/docs/blob/master/docs/mine-hnt/validators/monitor.mdx#monitor-a-validator) is the following:

```
alias miner="docker exec validator miner"
```

Therefore, I believe the current commands to be accurate. Have I misunderstood something @kellybyrd

lthiery commented:
I'm closing this for now @kellybyrd but feel free to re-open if I did not understand something.

kellybyrd commented:
The "bug" is on https://github.com/helium/docs/blob/master/docs/mine-hnt/validators/validator-deployment-guide.mdx

That document sets an alias called "validator", then later instructs the user to use  aliases called "val" or "miner". My commit is only relevant to this single doc.

I can't reopen the PR myself, it seems

kellybyrd committed:
Change back to using "miner" for the alias

After discussion on discord, decided to change the alias to miner, and update the sample commands. This will make it consistent across all the docs.
